src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -105,6 +106,13 @@
     String getWorldName();
 
     /**
+     * Gets the directory the files for this world are located.
+     *
+     * @return The directory the files are located in
+     */
+    File getWorldDirectory();
+
+    /**
```

Comment:
If we asume the world has a folder. Can we add methods for regions files as well?
Because both ways have been rejected last time this was discussed. For the above reasons.

> Just an additional two-cents, however - If we're going to go this route of allowing the potential for world-storage that isn't flat-file, that means whatever service defines the world location would also need a standardized way for storing additional data provided by plugins.

The plugin could store it in its own folder or database. It is not required to save it in the worlds folder. I really dont like plugins that randomly polute the world folder.